hoomd/simulation.py

```diff
@@ -63,13 +64,43 @@ def timestep(self):
 
     @timestep.setter
     def timestep(self, step):
-        if step < 0:
-            raise ValueError("Timestep must be positive.")
+        if int(step) < 0 or int(step) > 2**64 - 1:
```

Comment:
Should we define a module-level constant `TIMESTEP_MAX = 2**64 - 1` and `SEED_MAX = 2**16 - 1` and re-use those values in this file?

Reply:
done